docs/conf.py

```diff
@@ -20,15 +20,19 @@
 
 
 def init_tutorials():
-    if not "conf.py" in os.listdir():
+    if "conf.py" not in os.listdir():
         raise FileNotFoundError("Wrong directory...")
 
-    if not TUTREPO in os.listdir():
+    if TUTREPO not in os.listdir():
         command = f"git clone {TUTURL}"
         print(command)
         subprocess.call(command, shell=True)
-    if not TUTREPO in os.listdir():
+    if TUTREPO not in os.listdir():
         raise FileNotFoundError("Failed to clone pyaerocom-tutorials repo into pyaerocom/docs")
```

Comment:
the `if` and `raise` is no longer needed if `git pull` raises an exception

Reply:
It will raise `fatal`, which is not a python error but gets the point across.